jvm/core/src/main/kotlin/com/intuit/playerui/core/player/HeadlessPlayer.kt

```diff
@@ -77,6 +78,8 @@ public constructor(
 
     override val hooks: Hooks by NodeSerializableField(Hooks.serializer(), NodeSerializableField.CacheStrategy.Full)
 
+    public val constantsController: ConstantsController by NodeSerializableField(ConstantsController.serializer(), NodeSerializableField.CacheStrategy.Full)
```

Comment:
We should make this a part of the base `Player` class too (which would then require the `override` keywords):
https://github.com/player-ui/player/blob/bfd6a11a8d6c7138daec4724a8f08e9d9c4b370b/jvm/core/src/main/kotlin/com/intuit/playerui/core/player/Player.kt#L31-L32